app/server/src/main/scala/org/bitcoins/server/WalletRoutes.scala

```diff
@@ -256,6 +256,29 @@ case class WalletRoutes(wallet: AnyDLCHDWalletApi)(implicit
           }
       }
 
+    case ServerCommand("getdlcoffer", arr) =>
+      GetDLCOffer.fromJsArr(arr) match {
+        case Failure(exception) =>
+          complete(Server.httpBadRequest(exception))
+        case Success(GetDLCOffer(tempContractId)) =>
+          complete {
+            for {
+              dlcOpt <- wallet.findDLCByTemporaryContractId(tempContractId)
+              dlc = dlcOpt.getOrElse(
+                throw new IllegalArgumentException(
+                  s"Cannot find a DLC with temp contact ID $tempContractId"))
+              offerOpt <- wallet.getDLCOffer(dlc.dlcId)
+              offer = offerOpt.getOrElse(
+                throw new IllegalArgumentException(
+                  s"Cannot find an offer with for DLC ID ${dlc.dlcId}"))
+            } yield {
+              val tlv = offer.toTLV
```

Comment:
We return the `LnMessage(tlv)` here. We should probably be consistent and return the same format here too.

https://github.com/bitcoin-s/bitcoin-s/blob/9cfe65f1cca8e3a7df78e0c1d07866bfbe9a28da/app/server/src/main/scala/org/bitcoins/server/WalletRoutes.scala#L348